### ISSUE TITLE
Reformatting docstrings and comments

### ISSUE THREAD

**Issue by tschijnmo** [google/yapf#279]
By running the current 0.10.0 version of yapf on

```Py
def func(a):
    """Do something awesome.

    This is a long long long description of the awesome work that is done by this function.  Despite being very detailed, it runs over the column limit.
    """
```

we basically cannot get any formatting of the docstring.  Could you please help me if it is a feature that is currently absent from yapf?  If it could be reformatted into something like,

```Py
def func(a):
    """Do something awesome.

    This is a long long long description of the awesome work that is done by
    this function.  Despite being very detailed, it runs over the column limit.
    """
```

quite a lot of something like `gq` in vim could be saved.  If it could reformat the docstring in the default Sphinx, numpy, or Google format, probably it could be even more helpful.

**Comment by JorisE** [comment]
I would also appreciate the option to have a different maximum line length for comments.

**Comment by JorisE** [comment]
@tschijnmo: I just ran into https://github.com/myint/docformatter. That might help?

**Comment by carlthome** [comment]
Yes please. YAPF doesn't seem to reformat docstrings at all. Intended behavior?

**Comment by gwelymernans** [comment]
I've done this internally, and it's a mess. There are so many conditions that are needed. The best way would be to create a `lib2to3` fixer. If someone wanted to offer a patch, I would consider it.

**Comment by coxley** [comment]
I assume this goes for reflowing comments as well? Would be handy for sure.

**Comment by carlthome** [comment]
How about just piggybacking off of https://github.com/myint/docformatter?

**Comment by e-kwsm** [comment]
How about doctest?

```python
def f():
    """
    >>> 1+2  # -> 1 + 2
    3
    """
